### PR TITLE
Final adjustments to make it into a package

### PR DESCRIPTION
**Pull request by ilyasst**
There are the adjustments necessary to install it into a package.
Also, the README.md is missing a section that explains how to install the package and the necessary dependencies (I have added a requirements.txt for this).

### CODE REVIEW

File: WordAnalysis/wordanalysis_package/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+pandas
+nltk
+matplotlib
+geonamescache
+folium
```
Comment (Valirou) [inline WordAnalysis/wordanalysis_package/requirements.txt:5]: Are there some missing? Shouldn't it be : 
pandas
nltk
collections
matplotlib
math
geonamescache
folium
numpy
operator
re

If not, why?

File: WordAnalysis/wordanalysis_package/main_word_analysis.py [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+
+# Read the database file
+raw_data = pd.read_csv("../DataBase_20210522.csv", sep=";")
+
+# Return the data frame of raw_data
+raw_data.head
+
+# Simplified the data frame
+raw_data_obj = wa.RawDf(raw_data)
```
Comment (Valirou) [inline WordAnalysis/wordanalysis_package/main_word_analysis.py:22]: Could probably use the class WOSdf and call the attributes later (columns).

File: WordAnalysis/wordanalysis_package/main_word_analysis.py [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+art_refined_by_score = pd.concat([clean_data['Score'], raw_data_simplified['Article Title'], clean_data['UT (Unique WOS ID)']], axis=1)
+art_sorted_by_score_df = pd.DataFrame(art_refined_by_score).sort_values(by='Score', ascending=False)
+# Writing the file
+art_sorted_by_score_df.to_csv('../Results/ArticlesSortedByScore.csv', sep=';')
+
+# Writing a csv file with the mention materials sorted by most common
+material_df.to_csv('../Results/MaterialsSortedByMostCommon.csv', sep=';')
+
+# Writing a csv file with articles about blend material
+art_with_blend.to_csv('../Results/ArticlesAboutBlend.csv', sep=';')
```
Comment (Valirou) [inline WordAnalysis/wordanalysis_package/main_word_analysis.py:132]: Should we add all the tests on word analysis in FunctionTest and also use TestManager (https://pypi.org/project/TestManager/)?
As I understand it, we are using main_word_analysis as the test file (not main code outside of the package that uses this package). Thus, main replaces the tests folder according to the PyPA tutorial (https://packaging.python.org/tutorials/packaging-projects/).

File: WordAnalysis/wordanalysis_package/README.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
+# README
+
+wordanalysis package contains all the functions used in the main code. The main code is also use as a test file this is why it is in the package folder : [main_word_analysis.py](https://github.com/ets-lipec/rjs_litterature_mining/blob/convertopackage/WordAnalysis/main_words_analysis.py)
```
Comment (Valirou) [inline WordAnalysis/wordanalysis_package/README.md:3]: Links are to be changed.

**Review by Valirou — COMMENTED** [review]
requirements incompleted?
README update with requirements and how to install (links to be change)

File: WordAnalysis/wordanalysis_package/requirements.txt (hunk 1 above)
Comment (ilyasst) [reply WordAnalysis/wordanalysis_package/requirements.txt:5]: I started from a completely fresh virtual environment (I had no pre-installed packages) and tried running the code. The code then required a package, I added that package to the requirements, installed it; then tried running wordanalysis again, it failed and required another package, I added that package to the requirements, installed it and kept doing this until the code ran.
I am going to try to guess why some packages were not in requirements.txt using this method:

- collections: comes with Python 3.6, part of the standard python library
- math: comes with Python, part of the standard python library
- numpy: was most likely automatically installed with pandas so I did not notice. However, it should be added to the requirements.txt
- operator: standard python library
- re: standard python library

How do I know they're part of the standard python library?
Google "python3 re", if the first results is the official python doc https://docs.python.org/3/library/re.html then it's part of the standard library.

File: WordAnalysis/wordanalysis_package/main_word_analysis.py (hunk 3 above)
Comment (ilyasst) [reply WordAnalysis/wordanalysis_package/main_word_analysis.py:132]: We won't follow exactly all the PyPi guidelines, for now, our main goal is to be able to install and run the code locally.

**Pull request merged**